### ISSUE TITLE
Issues with using xmlrunner, AttributeError: 'XMLTestProgram' object has no attribute 'warnings'

### ISSUE THREAD

**Issue by srisuman** [xmlrunner/unittest-xml-reporting#190]
We are using the following command to run some tests.

 python -m xmlrunner discover -p "testBVT_*"

This results in the following error
Traceback (most recent call last):
File "/usr/lib64/python2.7/runpy.py", line 174, in _run_module_as_main
"_main_", fname, loader, pkg_name)
File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
exec code in run_globals
File "/usr/lib/python2.7/site-packages/xmlrunner/_main_.py", line 19, in <module>
XMLTestProgram(module=None)
File "/usr/lib/python2.7/site-packages/xmlrunner/runner.py", line 124, in _init_
super(XMLTestProgram, self)._init_(*args, **kwargs)
File "/usr/lib64/python2.7/unittest/main.py", line 95, in _init_
self.runTests()
File "/usr/lib/python2.7/site-packages/xmlrunner/runner.py", line 142, in runTests
warnings=self.warnings,
AttributeError: 'XMLTestProgram' object has no attribute 'warnings'

Could you please look into the issue.

**Comment by arunrag** [comment]
Please note that this has been working on our setup till 15th March (last time we ran the test suite).  Yesterday (20th March) started getting the error.

**Comment by dinio31** [comment]
The same here with latest version (2.4.0). Needed to revert to 2.3.2

```
>pip uninstall unittest-xml-reporting
>pip install unittest-xml-reporting==2.3.2

```

**Comment by arunrag** [comment]
Thank you for the suggestion. We were on Fedora Linux when we encountered the issue. Reverting to 2.3.2 helped to unblock us.

**Comment by dnozay** [comment]
try master. there is a fix that hasn't been released yet

**Comment by dinio31** [comment]
Thanks @dnozay. Tried with v2.5.0 which includes the fix you mentioned.

The error has moved to another place:

```
Traceback (most recent call last):
  File "c:\python2715\Lib\runpy.py", line 174, in _run_module_as_main
"__main__", fname, loader, pkg_name)
  File "c:\python2715\Lib\runpy.py", line 72, in _run_code
exec code in run_globals
  File "C:\xxxxxx\lib\site-packages\xmlrunner\__main__.py", line 19, in <module>
XMLTestProgram(module=None)
  File "C:\xxxxxx\lib\site-packages\xmlrunner\runner.py", line 125, in __init__
super(XMLTestProgram, self).__init__(*args, **kwargs)
  File "c:\python2715\Lib\unittest\main.py", line 95, in __init__
self.runTests()
  File "C:\xxxxxx\lib\site-packages\xmlrunner\runner.py", line 177, in runTests
self.testRunner = self.testRunner(**kwargs)
  File "C:\xxxxxx\lib\site-packages\xmlrunner\runner.py", line 22, in __init__
super(XMLTestRunner, self).__init__(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'warnings'

```

**Comment by srisuman** [comment]
@dnozay I tried the master as well and ran into the same issue.

**Comment by dnozay** [comment]
fixed in 2.5.1
